src/scss/components/ec-grid/ec-grid.scss

```diff
@@ -0,0 +1,92 @@
+$gutters: 24px;
```

Comment:
These two variables are in global scope so we should give them more unique name, e.g.
`$ec-grid-gutters` and `$ec-grid-number-of-columns`

Reply:
This hasn't been resolved